src/main/fc/fc_core.c

```diff
@@ -649,39 +649,67 @@ void processRx(timeUs_t currentTimeUs)
        This is needed to prevent Iterm winding on the ground, but keep full stabilisation on 0 throttle while in air
        Low Throttle + roll and Pitch centered is assuming the copter is on the ground. Done to prevent complex air/ground detections */
-    if (FLIGHT_MODE(MANUAL_MODE) || !ARMING_FLAG(ARMED)) {
-        /* In MANUAL mode we reset integrators prevent I-term wind-up (PID output is not used in MANUAL) */
-        pidResetErrorAccumulators();
+
+    if (!ARMING_FLAG(ARMED)) {
+        DISABLE_STATE(ANTI_WINDUP_DEACTIVATED);
     }
-    else if (STATE(FIXED_WING_LEGACY) || rcControlsConfig()->airmodeHandlingType == STICK_CENTER) {
-        if (throttleStatus == THROTTLE_LOW) {
-            if (STATE(AIRMODE_ACTIVE) && !failsafeIsActive() && ARMING_FLAG(ARMED)) {
-                rollPitchStatus_e rollPitchStatus = calculateRollPitchCenterStatus();
 
-                // ANTI_WINDUP at centred stick with MOTOR_STOP is needed on MRs and not needed on FWs
-                if ((rollPitchStatus == CENTERED) || (feature(FEATURE_MOTOR_STOP) && !STATE(FIXED_WING_LEGACY))) {
-                    ENABLE_STATE(ANTI_WINDUP);
-                }
-                else {
-                    DISABLE_STATE(ANTI_WINDUP);
-                }
-            }
-            else {
-                DISABLE_STATE(ANTI_WINDUP);
-                pidResetErrorAccumulators();
-            }
-        }
-        else {
-            DISABLE_STATE(ANTI_WINDUP);
-        }
-    } else if (rcControlsConfig()->airmodeHandlingType == THROTTLE_THRESHOLD) {
+    // In MANUAL mode we reset integrators prevent I-term wind-up (PID output is not used in MANUAL)
+    if (FLIGHT_MODE(MANUAL_MODE) || !ARMING_FLAG(ARMED)) {
         DISABLE_STATE(ANTI_WINDUP);
-        //This case applies only to MR when Airmode management is throttle threshold activated
-        if (throttleStatus == THROTTLE_LOW && !STATE(AIRMODE_ACTIVE)) {
-            pidResetErrorAccumulators();
-        }
+        pidResetErrorAccumulators();
     }
-
+    else if (rcControlsConfig()->airmodeHandlingType == STICK_CENTER) {
+        if (throttleStatus == THROTTLE_LOW) {
+             if (STATE(AIRMODE_ACTIVE) && !failsafeIsActive()) {
+                 rollPitchStatus_e rollPitchStatus = calculateRollPitchCenterStatus();
+                 if ((rollPitchStatus == CENTERED) || (feature(FEATURE_MOTOR_STOP) && !STATE(FIXED_WING_LEGACY))) {
+                     ENABLE_STATE(ANTI_WINDUP);
+                 }
+                 else {
+                     DISABLE_STATE(ANTI_WINDUP);
+                 }
+             }
+             else {
+                 DISABLE_STATE(ANTI_WINDUP);
+                 pidResetErrorAccumulators();
+             }
+         }
+         else {
+             DISABLE_STATE(ANTI_WINDUP);
+         }
+    }
+    else if (rcControlsConfig()->airmodeHandlingType == STICK_CENTER_ONCE) {
+        rollPitchStatus_e rollPitchStatus = calculateRollPitchCenterStatus();
```

Comment:
@Airwide let's combine this line with the same previous as 
`const rollPitchStatus_e rollPitchStatus = calculateRollPitchCenterStatus();` computed before this if condition

Reply:
Done!